topeft/modules/corrections.py

```diff
@@ -15,7 +15,7 @@
 import correctionlib
 from coffea.jetmet_tools import JECStack, CorrectedMETFactory
 ### workaround while waiting the correcion-lib integration will be provided in the coffea package
-from topcoffea.modules.CorrectedJetsFactory import CorrectedJetsFactory
+from topcoffea.modules import CorrectedJetsFactory
```

Comment:
@bryates this approach triggered this error message:
```
 Traceback (most recent call last):
  File "/usr/share/miniconda/envs/coffea-env/lib/python3.9/site-packages/coffea/processor/executor.py", line 1654, in _work_function
    out = processor_instance.process(events)
  File "/home/runner/work/topeft/topeft/analysis/topeft_run2/analysis_processor.py", line 383, in process
    cleanedJets = ApplyJetCorrections(year, corr_type='jets').build(cleanedJets, lazy_cache=events_cache)
  File "/home/runner/work/topeft/topeft/topeft/modules/corrections.py", line 1127, in ApplyJetCorrections
    return CorrectedJetsFactory(name_map, jec_stack)
TypeError: 'module' object is not callable
```

Should I revert the commit then?

Reply:
Sure, I guess if your other fix in TopCoffea to rename the class worked, you can undo this change.